hack/deploy-logging.sh

```diff
@@ -420,26 +410,26 @@ else
 fi
 
 if [ "${LOGGING_DEPLOY_MODE:-install}" = install ] ; then
-    CLO_DIR=${CLO_DIR:-$GOPATH/src/github.com/openshift/cluster-logging-operator}
-    if [ ! -d $CLO_DIR ] ; then
-        CLO_DIR=$ARTIFACT_DIR/clo
-        get_operator_files $CLO_DIR cluster-logging-operator ${CLO_REPO:-openshift} ${CLO_BRANCH:-master}
+    CLO_DIR=$GOPATH/src/github.com/openshift/cluster-logging-operator
+    EO_DIR=$GOPATH/src/github.com/openshift/elasticsearch-operator
+    if [ ! -d $GOPATH/src/github.com/openshift ] ; then
```

Comment:
unnecessary `if` check. `mkdir` will noop if the dir already exists

Reply:
mkdir will fail if dir already exists. `$?` is non zero
the script sets `set -euo pipefail` which makes script to exit if any operation fails.

Reply:
Not when you use the `-p` flag

```
$ mkdir -p $PWD
$ echo $?
0
```